chunked_stream/lib/src/read_chunked_stream.dart

```diff
@@ -32,7 +32,7 @@ import 'dart:async' show Stream, Future;
 /// ```
 Future<List<T>> readChunkedStream<T>(
   Stream<List<T>> input, {
-  int maxSize,
+  int? maxSize,
 }) async {
   ArgumentError.checkNotNull(input, 'input');
```

Comment:
We can probably remove checks like this all over the code, but that's just cleanup doesn't have to happen here..

I'll merge once I get CI working again :D